.gitignore

```diff
@@ -1,2 +1,3 @@
-vendor/
-composer.lock
+vendor/
+composer.lock
+/nbproject/
```

Comment:
nbproject should not be there. You should ignore it locally instead as it is specific to your setup: https://help.github.com/articles/ignoring-files/